circular_linked_list/circular_linked_list.cpp

```diff
@@ -0,0 +1,45 @@
+#include<iostream>
+
+using namespace std;
+
+struct node{
+    int data;
+    node* next;
+};
```

Comment:
Weird number of spaces at line-start.  Are you using a 2-space indent?  [whitespace/indent] [3]

*Origin: CPPLintBear, Section: `all.cpp`.*